test/checkout_test.rb

```diff
@@ -0,0 +1,87 @@
+require "test_helper"
+
+class CheckoutTest < Test::Unit::TestCase
+  def setup
+    super
+    ShopifyAPI::VaultSession.format = ShopifyAPI::VaultSessionFormat
```

Comment:
Because I need to avoid this: https://github.com/Shopify/shopify_api/blob/master/test/test_helper.rb#L33-L39